odoo/addons/base/models/ir_model.py

```diff
@@ -1980,8 +1981,25 @@ def delete(records):
         # remove models
         delete(self.env['ir.model'].browse(model_ids))
 
+        # attempt to remove undeletable model data that may have become deletable again because
+        # of the tables being dropped just above
```

Comment:
```suggestion
        # sort out which undeletable model data may have become deletable again because
        # of records being cascade-deleted or tables being dropped just above
```